src/CapabilitiesUtil/CapabilitiesUtil.ts

```diff
@@ -107,6 +107,10 @@ class CapabilitiesUtil {
       getFeatureInfoUrl = _get(wmsGetFeatureInfoConfig, 'DCPType.HTTP.Get.OnlineResource.href');
     }
 
+    if (!(layersInCapabilities instanceof Array)) {
```

Comment:
You could have used `Array.isArray` instead of the instanceof check. Doesn't matter in that case but should be preferd IMHO.